CPP11.md

```diff
@@ -874,6 +875,25 @@ auto handle = std::async(std::launch::async, foo);  // create an async task
 auto result = handle.get();  // wait for the result
 ```
 
+### std::begin/end
+`std::begin` and `std::end` free functions were added to return begin and end iterators of a container generically. These functions also work with raw arrays which do not have begin and end member functions.
+
+```c++
+template <typename T>
+int CountTwos(const T& container) {
+  return count_if(std::begin(container), std::end(container), [](int item) {
```

Comment:
Use `std::count_if`.

Also, even though we are just counting on `int` elements, I think we should still make the lambda parameter a `auto&` to keep the code as generic as we can. e.g. Think of if we were given a container with custom objects.

Reply:
This is a c++11 example code. Generic lambda parameters are introduced in C++14.

Reply:
You are correct! Good catch.